cmd/promtool/main.go

```diff
@@ -133,7 +133,11 @@ func main() {
 	dumpPath := tsdbDumpCmd.Arg("db path", "Database path (default is "+defaultDBPath+").").Default(defaultDBPath).String()
 	dumpMinTime := tsdbDumpCmd.Flag("min-time", "Minimum timestamp to dump.").Default(strconv.FormatInt(math.MinInt64, 10)).Int64()
 	dumpMaxTime := tsdbDumpCmd.Flag("max-time", "Maximum timestamp to dump.").Default(strconv.FormatInt(math.MaxInt64, 10)).Int64()
-
+	importCmd := tsdbCmd.Command("create-blocks-from", "[Experimental] Import samples from input and produce TSDB blocks. Please refer to the storage docs for more details.")
```

Comment:
This and rules backfill aren't pure TSDB, as they involve PromQL and exposition format parsing. Maybe this should be top level?

Reply:
Hmm, perhaps. But then we would probably have to call it `create-tsdb-blocks-from`. Which is even longer.

With the `tsdb` command now part of `promtool`, perhaps we don't have to be that strict anymore? It's a command that interacts with TSDB blocks, as are the other subcommands of `tsdb`. We might even have an option in the future the reads in the output of `promtool tsdb dump`. From that perspective, `create-blocks-from` is the counterpart to `dump` and should therefore be a `tsdb` subcommand, too.